src/main/java/com/minecolonies/coremod/commands/GetColonyAndCitizen.java

```diff
@@ -0,0 +1,196 @@
+package com.minecolonies.coremod.commands;
+
+import com.minecolonies.coremod.colony.ColonyManager;
+import net.minecraft.command.ICommandSender;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+/**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)